ddtrace/monkey.py

```diff
@@ -58,7 +58,7 @@ def patch_all(**patch_modules):
 
     :param dict \**patch_modules: Override whether particular modules are patched or not.
 
-        >>> patch_all({'redis': False, 'cassandra': False})
+        >>> patch_all(**{'redis': False, 'cassandra': False})
```

Comment:
Here we can convert it as:
```python
patch_all(redis=False, cassandra=False)
```

since the API for `patch_all()` is to use kwargs (that's why you need to add `**` to the previous example)